src/worker.ts

```diff
@@ -119,28 +123,46 @@ export class XeusRemoteKernel {
     });
     try {
       await waitRunDependency();
-
+      if (globalThis.Module['async_init'] !== undefined) {
```

Comment:
You need to change this check, otherwise only `xeus-python` (which is currently the only kernel defining this `async_init`), will bootstrap the filesystem.

We should bootstrap the filesystem from the lock-file as soon as there is a FileSystem defined:

```suggestion
      if (globalThis.Module.FS !== undefined) {
```